### ISSUE TITLE
Config option to only replace a percentage of Vanilla Villages?

### ISSUE THREAD

**Issue by xFirefalconx** [jtorleonstudios/HUB-Better-Villages#24]
Hi, I've seen that there is currently no config for this mod. I wonder if it'd be possible to add a percentage value to leave a few villages vanilla, when possible. This way, players could be encouraged to upgrade vanilla villages inspired by this mod, but also to create more varieties. 
The version I would be looking for is 1.18.2 on Forge 40.2.0. (Modpack, that's why)

**Comment by jtorleonstudios** [comment]
not planned

maybe later

**Comment by Flexico** [comment]
I came here precisely to ask this! Your village design would feel even more special if only 1/3 or so villages were upgraded~

**Comment by jtorleonstudios** [comment]
Yes, I see, I've already thought about it but I haven't yet looked at the java or json code. I'm going to release an update soon and after I'm going to study this request.

I need to study the possibilities carefully, because I'm afraid of adopting the idea and creating problems with other mods.

**Comment by ZsoltMolnarrr** [comment]
I love this idea!

**Comment by raffle2** [comment]
Supporting this idea! Also thanks for all the work so far
